phabfive/paste.py

```diff
@@ -32,6 +35,48 @@ def _convert_ids(self, ids):
 
         return ids_list_int
 
+    def create_paste(
+        self, title=None, file=None, language=None, tags=None, subscribers=None
+    ):
+        """Wrapper that connects to Phabricator and creates paste.
+
+        :type title: str
+        :type file: str
+        :type language: str
+        :type tags: list
+        :type subscribers: list
+
+        :rtype: dict
+        """
+        text = None
+        tags = tags if tags else []
+        subscribers = subscribers if subscribers else []
+
+        with open(file, "r") as f:
+            text = f.read()
+
+        transactions = []
+        transactions_values = [
+            {"type": "title", "value": title},
+            {"type": "text", "value": text},
+            {"type": "language", "value": language},
+            # TODO: Create a function that vaildates tags' and 'subscribers' existence
+            {"type": "projects.add", "value": tags},
+            {"type": "subscribers.add", "value": subscribers},
+        ]
+        # Phabricator does not take None (empty list is ok for projects/subscribers) as a value, therefor only "type" that has valid value can be sent as an argument
+        for item in transactions_values:
+            if None not in item.values():
+                transactions.append(item)
+        try:
+            id_and_phid = self.phab.paste.edit(transactions=transactions)
+        except APIError:
+            raise PhabfiveDataException(
+                "No valid input, maybe unvalid tags/subscribers or other error"
```

Comment:
"Invalid input"

Is it possible to gather information from the `APIError` what kind of problem caused this? If so, then that should probably be passed through to `PhabfiveDataException`.

Reply:
I will then change to 
```
except APIError as a:
    raise PhabfiveDataException(a)
```
which results in:
```
~/dynamist/phabfive(feature/create-paste*) » phabfive paste create new file.txt --subscribers=testuser1235790
ERR-CONDUIT-CORE: Edges are not available for objects of type '????'!

~/dynamist/phabfive(feature/create-paste*) » phabfive paste create new file.txt --tags=qjk
ERR-CONDUIT-CORE: Edge transactions must have destination PHIDs as in edge lists (found key "qjk" on transaction of type "41").
```

Reply:
Do you have suggestion on how to remove `ERR-CONDUIT-CORE:`? :) `a.replace('ERR-CONDUIT-CORE:', '')`  does not apply

Reply:
`AttributeError: 'APIError' object has no attribute 'lstrip'`
`a` is not a string

Reply:
Used `replace()` because `lstrip()` weirdly strip more that what it is suppose to do.